### DISCUSSION TITLE
General game conventions and strategies

### DISCUSSION THREAD

**Discussion by paolotron** [paolotron/Drop-Hanabi#8] [Ideas]
Here we can share conventions and strategies for the hanabi game

**Comment by Alkio84** [comment]
useful conventions:
• Hints generally indicate play cards, and generally newer cards first.
• Hints “chain” on to other hints, e.g., if A hints to B a playable red 2 as red, then
B might infer that it is indeed the red 2, and then hint back to A a red 3 in A’s
hand as red, expecting A to believe it is a red 3 to play it after B plays its red 2.
• When discarding, discard provably useless cards, otherwise the oldest “unprotected” card.
• Hints about the oldest unprotected card “protect” that card, with many exceptions
where it instead means play.
• Hints about garbage cards indicate “protect” cards older than it.
• Deliberately discarding known playable cards signals to the partner that they have
a copy of that card (with enough convention-based specificity on location that they
may play it with absolutely no actual hints).
• In many cases, hints about cards that have already been hinted change the belief
about those cards in various ways such that the partner will likely change what
they do (in accordance with the broader heuristic that one usually should give a
hint if and only if they would like to change the partner’s future behaviour).
